Makefile

```diff
@@ -1055,72 +1060,47 @@ bench-addons-clean:
 	$(RM) -r benchmark/napi/function_call/build
 	$(RM) -r benchmark/napi/function_args/build
 
+.PHONY: lint-md-rollup
+lint-md-rollup:
+	cd tools/node-lint-md-cli-rollup/remark-preset-lint-node && npm up
+	cd tools/node-lint-md-cli-rollup && npm up
+	cd tools/node-lint-md-cli-rollup && npm run build-node
+
 .PHONY: lint-md-clean
 lint-md-clean:
-	$(RM) -r tools/remark-cli/node_modules
-	$(RM) -r tools/remark-preset-lint-node/node_modules
+	$(RM) -r tools/node-lint-md-cli-rollup/remark-preset-lint-node/node_modules
+	$(RM) -r tools/node-lint-md-cli-rollup/node_modules
 	$(RM) tools/.*mdlintstamp
 
-tools/remark-cli/node_modules: tools/remark-cli/package.json
-	@echo "Markdown linter: installing remark-cli into tools/"
-	@cd tools/remark-cli && $(call available-node,$(run-npm-ci))
-
-tools/remark-preset-lint-node/node_modules: \
-	tools/remark-preset-lint-node/package.json
-	@echo "Markdown linter: installing remark-preset-lint-node into tools/"
-	@cd tools/remark-preset-lint-node && $(call available-node,$(run-npm-ci))
-
 .PHONY: lint-md-build
-lint-md-build: tools/remark-cli/node_modules \
-	tools/doc/node_modules \
-	tools/remark-preset-lint-node/node_modules
-
-tools/doc/node_modules: tools/doc/package.json
-ifeq ($(node_use_openssl),true)
-	cd tools/doc && $(call available-node,$(run-npm-ci))
-else
-	@echo "Skipping tools/doc/node_modules (no crypto)"
-endif
-
-.PHONY: lint-md
-ifneq ("","$(wildcard tools/remark-cli/node_modules/)")
+lint-md-build: 
+	$(warning "Deprecated no-op target 'lint-md-build'")
```

Comment:
Why should this not trigger the rollup?

Reply:
It has different semantics, before it was necessary to run `lint-md-build` for `lint-md` to work. Now `lint-md-rollup` is more like `update-eslint.sh`. It needs to be run once in a while, and the results should be checked in.
Also It is currently run in CI for every run, and that is unnecessary.